.gitignore

```diff
@@ -1,30 +1,8 @@
 
-cuda-stream
-ocl-stream
-omp-stream
-acc-stream
-raja-stream
-kokkos-stream
-std-stream
-sycl-stream
-hip-stream
-
-*.o
-*.bc
-*.sycl
-*.tar
-*.gz
-*.a
-
-KokkosCore_config.*
-
-.DS_Store
-
-Makefile
 
 build/
 cmake-build-*/
 CMakeFiles/
 .idea/
 .vscode/
-.directory
```

Comment:
What is this for?

Reply:
KDE's Dolphin, and maybe some other file managers leave that, similar to `.DS_store`